src/packages/engine-core/src/undici.ts

```diff
@@ -4,10 +4,13 @@ import { Pool } from 'undici'
 export class Undici {
   private pool: any
   private closed = false
-  constructor(url: string) {
+  constructor(url: any, moreArgs?: any) {
     this.pool = new Pool(url, {
```

Comment:
Just found out about idleTimeout, not sure if we should change the default 🤔 

```
idleTimeout: Number, the timeout after which a socket without active requests will time out. Monitors time between activity on a connected socket. This value may be overriden by keep-alive hints from the server. Default: 4e3 milliseconds (4s).
```

Reply:
Ok we can keep the default there for now, will remove that :)

Reply:
Actually you didn't change it I think, but I was just thinking should we? 🤔